datalab/datalab_session/data_operations/fits_output_handler.py

```diff
@@ -0,0 +1,62 @@
+import tempfile
+import numpy as np
+from astropy.io import fits
+
+from datalab.datalab_session.file_utils import create_jpgs
+from datalab.datalab_session.s3_utils import save_fits_and_thumbnails
+
+
+class FITSOutputHandler():
+  """A class to handle FITS output files and create jpgs.
+  
+  Class handles the creation of Datalab output for developers.
+  The class inits with a cache_key and data, and creates a FITS file with the data.
+  The FITS file is then saved to the cache and the large and small jpgs are created.
+
+  Attributes:
+    dlab_id (str): The cache key for the FITS file.
```

Comment:
As a new programmer to this project, it would probably confuse me to have dlab_id map to a cache key. Can we make this name more meaningful/descriptive?